### ISSUE TITLE
errors in visualisation in webgl_loader_gltf_transmission with safari 15.4 on M1/iOS devices

### ISSUE THREAD

**Issue by robertoranon** [mrdoob/three.js#23730]
In Safari 15.4 on M1 machines, opening the webgl_loader_gltf_transmission example and moving the camera around results in parts of the glb model missing

The same machine does not exhibit the issue when using Chrome

**To Reproduce**

1. Go to https://threejs.org/examples/#webgl_loader_gltf_transmission
2. Move the camera around
3. parts of the olive dish are missing

**Expected behavior**

**Screenshots**

<img width="1080" alt="Screenshot 2022-03-16 at 12 40 01" src="https://user-images.githubusercontent.com/6480367/158582110-6d76b252-ed29-4325-9619-45b079602bc9.png">

**Platform:**

 - Device: Macbook PRO M1 Max 14"
 - OS: macOS 12.3
 - Browser: Safari
 - Three.js version: r138

**Comment by Mugen87** [comment]
Do you mind testing how the example renders with Firefox? 

If the glitch only happens with Safari, it's probably a Webkit issue and should reported at: https://bugs.webkit.org/

**Comment by awong1900** [comment]
It is really big problem for me, PBR models is all broken.  I got a solution from google, need closed antialias. 
```
 renderer = new WebGLRenderer({
    antialias: false,
  });
````
I still wait a nice solution for this issue, wating...
If anyone have another

**Comment by vphilot** [comment]
Running into the same issue here with frames overlapping on Safari M1. @awong1900 solution's works great but my model looks terrible now lol

**Comment by mrdoob** [comment]
Hopefully Apple fixes this in iOS 15.5 🤞

**Comment by robertoranon** [comment]
It should be fixed in webkit: https://bugs.webkit.org/show_bug.cgi?id=238171

**Comment by uoon-dev** [comment]
@awong1900 Solutions works great! The afterimage side effect of UnrealBloompass also solved temporary. Thank you.

**Comment by Mugen87** [comment]
Should be fixed with the recently published iOS 15.5.
